src/Sql/Sql/Auditing/Cmdlet/RemoveAzSqlDatabaseAuditPolicy.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Management.Automation;
```

Comment:
@bashahee please add a license header to this file

Reply:
Done.